src/Application/AnalyzeCommand.php

```diff
@@ -4,7 +4,6 @@
 
 namespace TwigStan\Application;
 
-use InvalidArgumentException;
```

Comment:
@ruudk You should decide if such class should be used or not and let php-cs-fixer doing the job
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/import/global_namespace_import.rst

Reply:
@raffaelecarelle Could you create another PR that configures `global_namespace_import` like this:
```
['import_classes' => true, 'import_constants' => false, 'import_functions' => false]
```
?

We only want to import classes. Constants and functions not.

Reply:
Done in https://github.com/twigstan/twigstan/pull/28